saleor/order/models.py

```diff
@@ -142,7 +137,7 @@ def get_delivery_total(self):
                    Price(0, currency=settings.DEFAULT_CURRENCY))
 
     def send_confirmation_email(self):
-        email = self.get_user_email()
+        email = self.user_email
```

Comment:
Not sure about this change. There is no way to change this email for logged in users.